packages/babel-core/src/transformation/file/file.js

```diff
@@ -64,6 +65,14 @@ export default class File {
   }
 
   set(key: mixed, val: mixed) {
+    if (key === "helpersNamespace") {
+      throw new Error(
+        "Babel 7.0.0-beta.56 has dropped support for the 'helpersNamespace' utility." +
+          "If you are using @babel/plugin-external-helpers you will need to use a newer " +
+          "version than the one you currently have installed.",
```

Comment:
This error message should suggest plugin authors to use `helperGenerator` instead.